packages/cyberstorm/src/components/PackageCard.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { ReactNode } from "react";
+import styles from "./componentStyles/PackageCard.module.css";
+import { MetaItem } from "./MetaItem";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faDownload,
+  faThumbsUp,
+  faHardDrive,
+} from "@fortawesome/free-solid-svg-icons";
+import { Tag } from "./Tag";
+
+const defaultImageSrc = "";
+
+export interface PackageCardProps {
```

Comment:
I'd suggest double checking which of these parameters are actually optional. ~~Nevermind, there's defaultProps.~~ Also the defaultProps include props that are required and don't have values for props that are optional.

Also when I worked on the old edition of these components, it was discussed with Mythic that the terms should try to match the terms used in backend code to make life easier for developers working with both code bases. E.g. should author be teamName oslt?

Also also maybe double check the types of the values returned by the backend, e.g. is there's no image, will `null` be returned? It'd be awkward to have to cast that to `undefined`. Some of the props that sound they'd be numbers are marked as strings.